include/boost/core/lightweight_test.hpp

```diff
@@ -527,6 +596,48 @@ inline void lwt_init()
     boost::detail::test_results();
 }
 
+class lwt_context
+{
+    boost::detail::lwt_context_frame frame_;
+    
+    // Disallow copy and movement
+    lwt_context(const lwt_context&) {};
+    lwt_context& operator=(const lwt_context&) { return *this; }
```

Comment:
Please remove the bodies here; the C++03 idiomatic way to disable copy and assign is to declare the members as private and not define them.

Reply:
Better use `BOOST_DELETED_FUNCTION`, if we intend to keep C++03 compatibility.